### DISCUSSION TITLE
Welcome to ggprism Discussions!

### DISCUSSION THREAD

**Discussion by csdaw** [csdaw/ggprism#4] [General]
## 👋 Welcome!
You can use Discussions to connect with myself and other users of `ggprism`. I hope that you:

  * Share ideas for new features or documentation (Ideas)
  * Ask questions (Q&A)
  * Show off your plots (Show and tell)
  * Welcome others and are respectful and open-minded

**Comment by Rothcopico** [comment]
Hello there!

I was wondering whether you would be able to provide an example of ggprism that uses a time series as a dataset? I can use the various examples with no problems. However, I cannot seem to manipulate the minor labels on an x axis using the functions. I'm not sure whether this is my error or something in the code. 

An example:

plot1 <- ggplot(data = df, aes(x = as.Date(Date), y = Value, color = Security)) + 
         geom_line(size = 1.1)+
         theme_prism()+
         theme(legend.position = "top")+
         scale_y_continuous(guide = "prism_minor") +
         guides(x = "prism_minor",
                     limits = c(as.Date("2020-01-04"),as.Date("2021-07-31")),
                     minor_breaks = seq(from = as.Date("2020-01-04"), to = as.Date("2021-07-31"), by = "1 months"))

In this example, the graph prints the default minor breaks between dates rather than the expected 1 month spacing. A similar thing seems to happen if I adjust the 'by' argument to 3 months etc. Could you provide any insight? Thanks very much.

**Reply by csdaw** [reply]
Hi there,

Here is how I would do it. I'm not sure of the structure of your `df` so I just borrowed some data from the R Graph Gallery. Hope this helps 😄 

``` r
library(ggplot2)
library(ggprism)
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union

# Dummy data
data <- data.frame(
  Date = as.Date("2021-07-31") - 0:364,
  Value = runif(365) + seq(-140, 224)^2 / 10000,
  Security = c(rep("A", 134), rep("B", 365-134))
)

ggplot(data, aes(x = Date, y = Value, colour = Security)) +
  geom_line(size = 1.1) + 
  theme_prism() + 
  theme(legend.position = "top") + 
  scale_y_continuous(guide = "prism_minor") + 
  scale_x_date(
    guide = "prism_minor", 
    date_minor_breaks = "1 month",
    limits = as.Date(c("2020-01-04", "2021-07-31"))
  )
```

![](https://i.imgur.com/ogtAymt.png)

<sup>Created on 2021-08-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>
